ejercicios/ejercicio-010/ejercicio.py

```diff
@@ -5,8 +5,8 @@
 
 
 def tercera_letra(palabra):
-    pass
-
+    letra = palabra[2]
+    return letra
```

Comment:
Aprovado, podrìa haber sido mas corto e igualmente claro

```suggestion
    return palabra[2]
```